tests/hello_world.bats

```diff
@@ -6,15 +6,211 @@
 }
 
 @test "Test simple environment variable" {
-  [ "${PROGRAM}" == "hello_world" ]
+  echo "Val: ${SIMPLE_VAL}"
```

Comment:
The tests make me more confident that this works, than does looking at the code :)

Reply:
And thus tests consumed us all, and it was good.